### ISSUE TITLE
How are the pretrained models structured and how can we access the structure?

### ISSUE THREAD

**Issue by Xenopaia** [tarrade/proj_multilingual_text_classification#5]
What happens exactly in the BERT layer?

How can we unfold the structure using keras?

What happens exactly in the TFBertForSequenceClassification version?

**Comment by tarrade** [comment]
one of the best visualization:

![image](https://user-images.githubusercontent.com/12021701/79760684-b6f62c00-8320-11ea-8d8e-dce189d72875.png)

I didn't find yet a way to see the full structure with keras but using pytorch:
![image](https://user-images.githubusercontent.com/12021701/79760795-ddb46280-8320-11ea-8583-f3b9e6c03083.png)
![image](https://user-images.githubusercontent.com/12021701/79760834-e86ef780-8320-11ea-8c26-f6d8e5a78766.png)

TFBertForSequenceClassification return the logits (you can path in the config how much class do you have )
